src/sass/components/_movie-modal.scss

```diff
@@ -1,9 +1,235 @@
+  }
+}
+.movie {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  @include mq(tablet) {
+    flex-direction: row;
+  }
+  &__description {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    gap: 20px;
+    margin: 0;
+    padding-left: 20px;
+    padding-right: 20px;
+
+    @include mq(tablet) {
+      padding: 40px 72px 40px 68px;
+    }
+    @include mq(desktop) {
+      padding: 40px 12px 111px 16px;
+    }
+  }
+  &__title {
+    font-family: $FF;
+    font-weight: var(--fw-500);
+    font-size: var(--fs-700);
+    line-height: var(--lh-300);
+    color: $font-color-dark;
+    text-transform: uppercase;
+  }
+}
+.movie-image {
+  border-radius: 5px;
+  width: 240px;
+  height: 357px;
+  border-radius: 5px;
+
+  @include mq(tablet) {
+    width: 264px;
+    height: 374px;
+  }
+  @include mq(desktop) {
+    width: 375px;
+    height: 478px;
+  }
+  &__delay {
+    display: flex;
+    justify-content: center;
+    padding-bottom: 20px;
+    padding-top: 48px;
+
+    @include mq(tablet) {
+      padding-top: 40px;
+      padding-left: 36px;
+    }
+    @include mq(desktop) {
+      padding-top: 40px;
+      padding-left: 12px;
+    }
+  }
+}
+
+.table {
+  margin-top: 20px;
+  width: 215px;
+
+  @include mq(desktop) {
+    width: 259px;
+  }
+  &__row {
+    font-family: $FF;
+    font-weight: var(--fw-500);
+    font-size: var(--fs-400);
+    line-height: var(--lh-500);
+    color: $font-color-info;
+    &-item {
+      display: inline-block;
+    }
+  }
+  &__title--votes {
+    padding-right: 41px;
+
+    @include mq(desktop) {
+      padding-right: 85px;
+    }
+  }
+  &__title--popularity {
+    padding-right: 53px;
+
+    @include mq(desktop) {
+      padding-right: 97px;
+    }
+  }
+  &__title--original-title {
+    padding-right: 39px;
+
+    @include mq(desktop) {
+      padding-right: 83px;
+    }
+  }
+  &__title--original-genre {
+    padding-right: 76px;
+
+    @include mq(desktop) {
+      padding-right: 120px;
+    }
+  }
+}
+
+.details {
+  font-size: var(--fs-400);
+  font-family: $FF;
+  font-weight: var(--fw-500);
+  line-height: var(--lh-500);
+  text-transform: uppercase;
+  color: $font-color-dark;
+  &__description {
+    margin-top: 8px;
+    font-family: $FF;
+    font-weight: var(--fw-500);
+    font-size: var(--fs-400);
+    line-height: var(--lh-600);
+  }
+}
+.grouped-buttons {
+  width: 110px;
+  height: 44px;
+  font-family: $FF;
+  font-weight: var(--fw-500);
+  font-size: var(--fs-400);
+  line-height: ar(--lh-500);
+  text-align: center;
+  text-transform: uppercase;
+  border-radius: 5px;
+  background-color: $bg-color-light;
+  border: 1px solid #000000;
+  cursor: pointer;
+  &:hover,
+  &:focus {
+    background-color: $bg-color-rating-primary;
+    color: $button-font-color-light;
+    border: none;
+  }
+  @include mq(tablet) {
+    width: 125px;
+  }
+
+  @include mq(desktop) {
+    width: 136px;
+  }
+  &__delay {
```

Comment:
Dodaj do tej klasy
`padding-bottom: 20px;`
Chodzi o to, aby przy mobilce, gdy zjeżdżasz na dół modal-a to lepiej, aby był jakiś lekki odstęp.

Reply:
done